.github/workflows/docker-build-test.yaml

```diff
@@ -179,6 +181,7 @@ jobs:
       PROFILE: performance
       BUILD_ADDL_TESTING_IMAGES: true
       TARGET_REGISTRY: ${{ needs.determine-docker-build-metadata.outputs.targetRegistry }}
+      WAIT_FOR_PERF_IMAGES: true
```

Comment:
this is kind of redundant given that the `PROFILE: performance` is already specified. Can we unify them?